scribejava-apis/src/main/java/com/github/scribejava/apis/PinterestAPI.java

```diff
@@ -0,0 +1,44 @@
+package com.github.scribejava.apis;
+
+import com.github.scribejava.core.builder.api.DefaultApi20;
+import com.github.scribejava.core.extractors.AccessTokenExtractor;
+import com.github.scribejava.core.extractors.JsonTokenExtractor;
+import com.github.scribejava.core.model.OAuthConfig;
+import com.github.scribejava.core.model.OAuthConstants;
+import com.github.scribejava.core.model.Verb;
+import com.github.scribejava.core.utils.OAuthEncoder;
+import com.github.scribejava.core.utils.Preconditions;
+
+public class PinterestApi extends DefaultApi20 {
```

Comment:
non compilable class
it's name 'PinterestApi'
while file name is 'PinterestAPI'

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.3:compile (default-compile) on project scribejava-apis: Compilation failure
[ERROR] /tmp/scribejava/scribejava-apis/src/main/java/com/github/scribejava/apis/PinterestAPI.java:[12,8] class PinterestApi is public, should be declared in a file named PinterestApi.java


looks like you use Windows or some case-insensitive FS